src/browser/BrowserPasswordGeneratorWidget.cpp

```diff
@@ -0,0 +1,346 @@
+/*
```

Comment:
You seem to duplicate the whole PasswordGeneratorWidget. Isn't there a way to integrate the existing widget?